azext_edge/edge/providers/check/summary.py

```diff
@@ -58,7 +58,7 @@ def check_summary(
         ServiceCheck(
             svc=OPCUA_SERVICE,
             title="OPCUA",
-            target=OPCUA_API_V1.as_str(),
+            target="OPCUA",
```

Comment:
We'll need to establish checks relationship with connectors where opcua is one connector type. Since the functionality is in preview, I suppose this is fine.

Reply:
yeah this will be a future check change, I can add a TODO here

Reply:
probably in next pr since this going to reset the approval😋